### DISCUSSION TITLE
Recommended boilerplate for Next.js

### DISCUSSION THREAD

**Discussion by mxcdh** [vercel/next.js#61456] [Help]
### Summary

I want to build a Next.js marketplace for my side project. I'm searching for a boilerplate with good quality code and structure. At the moment, I found the following options:
- https://create.t3.gg/ - looks good
- https://github.com/ixartz/Next-js-Boilerplate
- https://github.com/steven-tey/precedent
- https://github.com/sly777/ran
Which boilerplate do you recommend? I prefer open source resources.

**Comment by jared-christensen** [comment]
I'm looking for a similar thing, I want an opinionated way to organize my project. I posted a similar question on Discord. https://nextjs-forum.com/post/1205557905844338760
